misc/docker/Dockerfile

```diff
@@ -1,23 +1,22 @@
 FROM busybox
 
 RUN set -x \
-    && wget -O- http://nixos.org/releases/nix/nix-1.9/nix-1.9-x86_64-linux.tar.bz2 | \
+    && wget -O- http://nixos.org/releases/nix/nix-1.10/nix-1.10-x86_64-linux.tar.bz2 | \
```

Comment:
Well in future i want to make automated builds on docker hub, so nix image will be allways up-to-date in that case i can't simply make substitutions.